### DISCUSSION TITLE
T-Pot sending DNS, HTTPS and HTTP requests

### DISCUSSION THREAD

**Discussion by I-Simon-I** [telekom-security/tpotce#1324] [Q&A]
Hi ! 

One of my T-Pot's trafic is passing through my IDS and out of curiosity I looked at its requests and such. To my surprise, I saw lots of requests even for the last 7 days and I was wondering, why does it send requests to "Amazon" or even "Tele Danmark" ?

![image](https://user-images.githubusercontent.com/116071161/236857692-72eb36d3-e911-4a32-af2e-6c7e35600d24.png)

![image](https://user-images.githubusercontent.com/116071161/236857766-7e3d141d-3396-4087-ad97-84b3238260ed.png)

Thx for your time !

**Comment by t3chn0m4g3** [comment]
Please check [here](https://github.com/telekom-security/tpotce#required-ports).

**Reply by I-Simon-I** [reply]
All of the above are outgoing trafic, that's why I'm asking. There is trafic on port 80 and 443 as mentionned in the documentation but what about the rest of them ?
